### ISSUE TITLE
Elgg fails : no settings.example.php

### ISSUE THREAD

**Issue by m-anish** [XSCE/xsce#805]
```
TASK: [elgg | Substitute our parameters in engine/settings.example.php] ******* 
fatal: [127.0.0.1] => input file not found at /opt/elgg/engine/settings.example.php or /opt/elgg/engine/settings.example.php

FATAL: all hosts have already failed -- aborting
```

**Comment by georgejhunt** [comment]
I discovered that elgg 2.1 requires php 5.5.   When I tried to install
php.5.5, Centos was not happy, and the console stuff didnt work.

I have tested a fix which uses 10.10.12. which is at
https://github.com/georgejhunt/xsce/tree/elgg

git pull ghunt elgg after you have added my repo

**Comment by m-anish** [comment]
PR#806 seems to fix this.

**Comment by m-anish** [comment]
Please close the issue once the PR is merged.

**Comment by jvonau** [comment]
merged PR#806 closing
